### ISSUE TITLE
Review and update SEO/OG tags

### ISSUE THREAD

**Issue by alxflw** [onflow/developer-portal#466]

**Comment by conwaykong** [comment]
SEO Elements to be optimized
- Title Tag
- Meta Description
- H1
- Canonical URL

Open Graph Tags
- OG:URL
- OG:TItle
- OG:Description
- OG: Image

**Comment by alxflw** [comment]
@10thfloor please take a look! we'd also like to understand the page types we have on the portal, so that there could be tags optimized for there. right now, it seems like there are at least two:
- hard-coded landing pages
- markdown-base / internal pages

But, I assume we implemented a few more page types, so input here would be useful 🙏

**Comment by 10thfloor** [comment]
These are the only 2 pages types.

**Comment by conwaykong** [comment]
[SEO Recommendations Spreadsheet](https://docs.google.com/spreadsheets/d/1VdWq4_nhLcybMoU49jIKinlmMjtrbKT-DCOuNrzM0_Y/edit?usp=sharing)

The spreadsheet above includes recommendations for the following
- Title Tag
- Meta Description
- H1 Title
- Canonical URL

Note: Duplicate URLs were discovered upon my initial page crawl (designated on the spreadsheet in red). We'll want to 301 redirect those pages to the original URL.

Open Graph Tags will use elements from SEO elements
OG:URL = Canonical URL
OG:TItle = Title Tag
OG:Description = Meta Description
OG: Image = TBD (Option: Designate one image (ex. Flow Logo) for all Developer pages)

**Comment by 10thfloor** [comment]
This is currently blocked by resolving the route refactor changes. 
- https://github.com/onflow/next-docs-v1/issues/487
- PR:

**Comment by 10thfloor** [comment]
Unblocked!

**Comment by LaurenIke** [comment]
## Items to fix:
- sitemap.xml - change [[onflow.org](http://onflow.org/)](http://onflow.org) to [[flow.com](http://flow.com/)](http://flow.com) ✅
- canonical url is not defined ✅
- optimize title tag and description (Conway to create an excel - separate item in the table)
- stretch goal - crosslinking (currently not required)

**Comment by nialexsan** [comment]
https://github.com/onflow/docs/issues/255
